### DISCUSSION TITLE
Wegpunkte direkt verbinden

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#53] [Q&A] [answered]
Ich schreibe in deutsch. Ich hoffe das geht in Ordnung 
Gibt es die Möglichkeit bei der Planung 2 Wegpunkte mit
 " Luftlinie" zu verbinden wenn sich dort keine Straße befindet ?
LG.

**Comment by devemux86** [comment, marked as answer]
Hi @1maxi2, you can write in any language!

The straight line is currently available with BRouter 1.6.4 when select "BRouter (offline)" in "Settings | Routing | Routing".
It is not released yet on Google Play, you can install it from [here](https://github.com/devemux86/cruiser/discussions/15).

Long press on the waypoints or on the route line to set the straight line.
Or use the waypoints list in the routing menu.

Cruiser will be able to use the straight line with all routers in the future (whether they support it or not).

---

Hallo @1maxi2, Sie können in jeder Sprache schreiben!

Die gerade Linie ist derzeit mit BRouter 1.6.4 verfügbar, wenn Sie "BRouter (offline)" in "Einstellungen | Routing | Routing" auswählen.
Es ist noch nicht auf Google Play veröffentlicht, Sie können es von [hier](https://github.com/devemux86/cruiser/discussions/15) installieren.

Drücken Sie lange auf die Wegpunkte oder auf die Routenlinie, um die gerade Linie zu setzen.
Oder verwenden Sie die Wegpunktliste im Routing-Menü.

Cruiser wird in Zukunft in der Lage sein, die gerade Linie mit allen Routern zu verwenden (unabhängig davon, ob sie es unterstützen oder nicht).

**Reply by 1maxi2** [reply]
O.k. Danke für die Antwort. Habe bis jetzt kurviger verwendet welches in der letzten Ausführung fast perfekt war. 
Mit Brouter kenne ich mich nicht aus. Verwende Standart Navigation - Cruiser 
( Mapsfourge ? )  Aber kein Problem ich warte bis es verfügbar sein wird.
Lg.

**Reply by devemux86** [reply]
You can **import kurviger routes in Cruiser** and navigate offline.

Any rerouting only calculates the path from user location to next waypoint, the rest of the route remains unchanged.

---

Sie können **kurviger Routen in Cruiser importieren** und offline navigieren.

Bei der Umleitung wird nur der Weg vom Benutzerstandort zum nächsten Wegpunkt berechnet, der Rest der Route bleibt unverändert.

**Reply by devemux86** [reply]
<img src="https://user-images.githubusercontent.com/3484020/220752452-608aea5c-3f47-411e-b0b5-50c50d59a679.jpg" width="50%"/><img src="https://user-images.githubusercontent.com/3484020/220752463-2b79187e-713c-4ab8-82ee-bd92790d3b87.jpg" width="50%"/>

**Comment by WolfgangZ42** [comment]
![2023-02-22 21 36 22](https://user-images.githubusercontent.com/121806238/220754689-1b425fde-cca5-4b41-947f-693dd94478a3.png)  ![2023-02-22 21 36 59](https://user-images.githubusercontent.com/121806238/220754719-c98f365b-d86f-4e7a-80f0-5bf570fbbe03.png)
Before . . . . . . . . . . . . . . . . After

Es gibt zwischen (1) und (2) eine Straße, diese ist jedoch nur für den landwirtschaftlichen Verkehr freigegeben. 😄 
There is a road between (1) and (2), but it is only open to agricultural traffic. 😄
